src/components/navbar/index.tsx

```diff
@@ -94,6 +96,19 @@ const NavBar: React.FC<NavBarProps> = ({ tokens }) => {
       >
         Settings
       </Menu.Item>
+      <Menu.Item
+        onClick={() => {
+          if (asyncRequestIsComplete(tokens)) {
+            AuthClient.logout(tokens.result.refreshToken).then(() => {
+              localStorage.removeItem(LOCALSTORAGE_STATE_KEY);
+            });
```

Comment:
This should be done by `dispatch(logoutUser())`, we have a thunk which makes this request and clears the store.